src/PackageResolver.php

```diff
@@ -20,9 +20,19 @@ class PackageResolver
 {
     private const SYMFONY_VERSIONS = ['lts', 'previous', 'stable', 'next'];
 
+    /**
+     * @var array
+     */
     private static $aliases;
+
+    /**
+     * @var array
```

Comment:
`string[]` ?

(Also, I personally prefer the inlined version:

```php
/** @var string[] */
private static $versions;
```
, but may be only me)